docs/src/api.md

```diff
@@ -58,19 +58,15 @@ These functions return *distributions*.
 transition
 observation
 initialstate_distribution
+reward
 ```
 
 ### [Generative](@id generative_api)
 
-These functions should return *states*, *observations*, and *rewards*.
+These functions should return *states*, *observations*, and/or *rewards*.
```

Comment:
Is it worth putting in another line about what `gen` represents for people already intimately familiar with the previous framework? Or even just that `gen` means generate?